### DISCUSSION TITLE
Immobilize A Joint

### DISCUSSION THREAD

**Discussion by mallasalla** [visose/Robots#124] [Q&A] [answered]
Hi, thank you very much for the upload, is precious!
I'm working with an UR10, with an extruder fixed on the end. The movement of the last joint is dangerous because risks to let impact the extruder on the robot itself, is it possible to immobilize it, letting work the other joints? I'm trying by modify the .xml file but... mmmh  :S

**Comment by visose** [comment, marked as answer]
Hi, you can only control individually the joints by using joint targets. If you use Cartesian targets, make sure their orientation are set in a way that the tool doesn't collide with the robot.
